### DISCUSSION TITLE
Do I have to implement getpeername?

### DISCUSSION THREAD

**Discussion by kmc7468** [ANLAB-KAIST/KENSv3#149] [Project Q&A 2. Transport Layer (TCP)] [answered]
According to the PA#2 document, we don't have to implement `getpeername` system call.
However, when I executed `test-kens-handshake`, it required me to implement the system call.

So, do I have to implement it?

**Comment by Pusnow** [comment, marked as answer]
Sorry for the confusion. We omitted getpeername part in the PA2 document.

You should implement getpeername for PA2.
